cmd/register/register_util.go

```diff
@@ -509,8 +510,18 @@ func GetSerializeOutputFiles(ctx context.Context, args []string, archive bool) (
 		sort.Strings(finalList)
 		return finalList, "", nil
 	}
-
-	tempDir, err := ioutil.TempDir("/tmp", "register")
+    
+    os := runtime.GOOS
+    var tempDir string
+    var err error
+    if os == "windows" {
+        fmt.Println("Windows Operating System Detected")
+        tempDir, err = ioutil.TempDir("\tmp", "register")
```

Comment:
\t is a tab... I think you need to escape that
```suggestion
        tempDir, err = ioutil.TempDir("\\tmp", "register")
```

Reply:
Nice catch... I just fixed it and made the commit. Thanks